tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino

```diff
@@ -12,20 +15,14 @@ RUN /tmp/scripts/install_ubuntu.sh -p ${PYTHON_VERSION} -d EdgeDevice && \
 RUN apt update && apt install -y libnuma1 ocl-icd-libopencl1 && \
     rm -rf /var/lib/apt/lists/* /tmp/scripts
 
-ENV INTEL_OPENVINO_DIR /opt/intel/openvino_${OPENVINO_VERSION}
+ENV INTEL_OPENVINO_DIR /opt/intel/openvino/
 ENV LD_LIBRARY_PATH $INTEL_OPENVINO_DIR/runtime/lib/intel64:$INTEL_OPENVINO_DIR/runtime/3rdparty/tbb/lib:/usr/local/openblas/lib:$LD_LIBRARY_PATH
 ENV InferenceEngine_DIR $INTEL_OPENVINO_DIR/runtime/cmake
 ENV ngraph_DIR $INTEL_OPENVINO_DIR/runtime/cmake
 ENV IE_PLUGINS_PATH $INTEL_OPENVINO_DIR/runtime/lib/intel64
 ENV DEBIAN_FRONTEND=noninteractive
 ENV PATH /usr/local/gradle/bin:$PATH
 
-RUN cd /opt && mkdir -p intel && cd intel && \
```

Comment:
From security perspective, I'd prefer to do it in the old way.  The old code clearly shows what are included in the build image, and we can audit the introduced 3rd party dependencies. The new one is a black box to us.  

For example, last weekend I found the `install_openvino_dependencies.sh` script installs nlohmann json to the system's include folder, but the version of the library is much older than ORT needs, and it may accidently clash with the one in ORT's submodule .  If you only give me a black box, I wouldn't be able to find it out.

Reply:
Ok. They have actually opened a ticket to resolve the SSL Cert Issue. However, it being a holiday in Ireland, they did not have a resolution, so keeping in mind the urgency we have made this change. If you could wait a day or two we can get the Certificate issue resolved.

Reply:
Ok